{{cookiecutter.repo_name}}/Makefile

```diff
@@ -1,10 +1,12 @@
 .DEFAULT_GOAL := test
+NODE_BIN=./node_modules/.bin
 
 .PHONY: clean compile_translations dummy_translations extract_translations fake_translations help html_coverage \
 	migrate pull_translations push_translations quality requirements test update_translations validate
 
 help:
-	@echo "Please use \`make <target>' where <target> is one of"
+	@echo "Please use \`make <target>\` where <target> is one of"
+	@echo "  bundle                     build optimized JS bundle, compile CSS, and run collectstatic"
```

Comment:
This target doesn't exist. If it did, it seems it does the same thing as `static`.